packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -647,8 +648,12 @@ export const Webchat = forwardRef((props, ref) => {
   }
 
   useEffect(() => {
+    if (firstUpdate.current) {
+      firstUpdate.current = false
+      return
+    }
     if (webchatState.isWebchatOpen && props.onOpen) props.onOpen()
-    if (!webchatState.isWebchatOpen && props.onClose) {
+    if (!webchatState.isWebchatOpen && props.onClose && !firstUpdate.current) {
```

Comment:
do you need to check firstUpdate.current again?

Reply:
nope, not necessary, removed 👍